app/src/main/java/de/westnordost/streetcomplete/map/PinIcons.kt

```diff
@@ -59,5 +60,20 @@ import de.westnordost.streetcomplete.R
     when (map["tourism"]) {
         "information" -> return R.drawable.ic_pin_information
     }
+    if (map.containsKey("addr:housenumber") || map.containsKey("addr:housename") ||
+            map.containsKey("addr:conscriptionnumber") || map.containsKey("addr:streetnumber")
+        && !map.containsAnyKey("amenity", "craft", "healthcare", "leisure", "office", "shop", "tourism")){
+        return R.drawable.ic_none
+    }
     return null
 }
+
+fun getTitle(map: Map<String, String>): String? {
+    return if (map["name"] != null || map["brand"] != null) {
+        map["name"] ?: map["brand"]
+    } else if (map["addr:conscriptionnumber"] != null && map["streetnumber"] != null) {
+        map["addr:conscriptionnumber"] + "/" + map["streetnumber"]
```

Comment:
1. it's `addr:streetnumber` , not `streetnumber`
2. `addr:streetnumber` is optional

Reply:
1. argh! Sorry! I did know that really
2. Added an extra option to the if